src/core/json/genjson.ml

```diff
@@ -159,6 +158,12 @@ and generate_metadata ctx ml =
 	) ml in
 	jlist (generate_metadata_entry ctx) ml
 
+and generate_minimum_metadata ctx ml =
+	let ml = List.filter (fun (m,_,_) ->
+		m = Meta.Deprecated
```

Comment:
Not sure how I feel about hardcoding it, maybe a filter array could be added to the request params so the client can choose what it wants to receive.